modules/s3_hosting/outputs.tf

```diff
@@ -7,3 +7,8 @@ output "bucket_arn" {
   description = "ARN of the created S3 bucket"
   value       = aws_s3_bucket.client_assets.arn
 }
+
+output "cf_signed_downloads" {
+  description = "Does this require signed URL downloads?"
+  value       = var.cf_signed_downloads
```

Comment:
Mind calling this something like `cf_signing_enabled` to indicate that it's a boolean? The question mark in your comment indicates it but it would be nice to be able to tell from the variable name

Reply:
Yeah, for sure. I originally had it named something else and then renamed it to this input var for consistency.
It would be nice to go back and name it consistently across all repositories at some point.

Reply:
Yeah. Sorry, I'm trying to be more vigilant that usual about naming in this repo because it's fairly outward-facing and meant to be re-used often.